### DISCUSSION TITLE
4.0.2

### DISCUSSION THREAD

**Discussion by gasman** [wagtail/wagtail#9251] [Announcements]
* Update all images and sections of the Wagtail Editor's guide to align with the new admin interface changes from Wagtail 3.0 and 4.0 (Thibaud Colas)
 * Ensure all images in the documentation have a suitable alt text (Thibaud Colas)
 * Fix: Ensure tag autocompletion dropdown has a solid background (LB (Ben) Johnston)
 * Fix: Allow inline panels to be ordered (LB (Ben) Johnston)
 * Fix: Only show draft / live status tags on snippets that have `DraftStateMixin` applied (Sage Abdullah)
 * Fix: Prevent JS error when initialising chooser modals with no tabs (LB (Ben) Johnston)
 * Fix: Add missing vertical spacing between chooser modal header and body when there are no tabs (LB (Ben) Johnston)
 * Fix: Reinstate specific labels for chooser buttons (for example 'Choose another page', 'Edit this page' not 'Change', 'Edit') so that it is clearer for users and non-English translations (Matt Westcott)
 * Fix: Resolve issue where searches with a tag and a query param in the image listing would result in an `FilterFieldError` (Stefan Hammer)
 * Fix: Add missing vertical space between header and content in embed chooser modal (LB (Ben) Johnston)
 * Fix: Use the correct type scale for heading levels in rich text (Steven Steinwand)
 * Fix: Update alignment and reveal logic of fields’ comment buttons (Steven Steinwand)
 * Fix: Regression from Markdown conversion in documentation for API configuration - update to correctly use PEP-8 for example code (Storm Heg)
 * Fix: Prevent 'Delete' link on page edit view from redirecting back to the deleted page (LB (Ben) Johnston)
 * Fix: Prevent JS error on images index view when collections dropdown is omitted (Tidiane Dia)
 * Fix: Prevent "Entries per page" dropdown on images index view from reverting to 10 (Tidiane Dia)
 * Fix: Set related_name on user revision relation to avoid conflict with django-reversion (Matt Westcott)
 * Fix: Ensure the "recent edits" panel on the Dashboard (home) page works when page record is missing (Matt Westcott)
 * Fix: Only add Translate buttons when the `simple_translation` app is installed (Dan Braghis)
 * Fix: Ensure that `MultiFieldPanel` correctly outputs all child classnames in the template (Matt Westcott)
 * Fix: Remove over-eager caching on ModelAdmin permission checks (Matt Westcott, Stefan Hammer)

<hr /><em>This discussion was created from the release <a href='https://github.com/wagtail/wagtail/releases/tag/v4.0.2'>4.0.2</a>.</em>

**Comment by th3hamm0r** [comment]
@gasman Thanks for the release!
https://github.com/wagtail/wagtail/issues/9113 is now fixed on the code side, but since the new strings from https://github.com/wagtail/wagtail/commit/26b3752cd35913e5cc338ce5793d9ed664e37150 haven't been uploaded to transifex yet, those changes are mainly untranslated in 4.0.2. So to really fix #9113 in other languages, the new po files have to be synced with transifex to allow them to be translated. So I hope for a 4.0.3 which contains the translations 😉

**Reply by gasman** [reply]
I did upload the new strings to Transifex while preparing the 4.0.2 release (sorry I didn't have chance to do so in advance...) and they're showing up now, as far as I can see. There will probably be a 4.0.3 release in a week or two, once https://github.com/wagtail/wagtail/pull/9240 has been reviewed and merged.

**Reply by th3hamm0r** [reply]
Oh sorry, that's true, the new strings are already in transifex (I expected some untranslated strings, but this time other translators were faster 😅), so it was just too late for 4.0.2, but I'm ok with 4.0.3, thanks!
